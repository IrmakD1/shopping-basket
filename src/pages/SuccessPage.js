import React from 'react';
import Success from '../containers/Success'

const SuccessPage = () => {
  return (
      <div>
          <Success />
      </div>
  )
}

export default SuccessPage
