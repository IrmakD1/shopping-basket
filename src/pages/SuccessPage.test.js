import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import SuccessPage from './SuccessPage';

describe('BasketPage page', () => {
    let renderer;
    let result;
    beforeEach(() => {
      renderer = new ShallowRenderer();
      renderer.render(<SuccessPage />);
      result = renderer.getRenderOutput();
    });
    test('it should render', () => {
      expect(result).toMatchSnapshot();
    });
});