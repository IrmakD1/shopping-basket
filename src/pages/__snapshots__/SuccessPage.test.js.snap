// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BasketPage page it should render 1`] = `
<div>
  <UNDEFINED />
</div>
`;
